agent/agent.go

```diff
@@ -337,15 +294,9 @@ func (a *Agent) RecordManifest() {
 	a.ManifestOps = result
 }
 
-// WriteOutput renders the manifest and results of the diagnostics run and writes the compressed archive.
+// WriteOutput renders the manifest and results of the diagnostics run into Agent.tmpdir
```

Comment:
Nit:
```suggestion
// WriteOutput renders the manifest and results of the diagnostics run into Agent.TmpDir
```

Reply:
Just to be sure: your intent is to export the temporary directory field on the agent?

Reply:
Oh, I was just thinking that the name of the field wasn't "tmpdir," but it's "TmpDir." Apologies if I misunderstood!

Reply:
This is a bit weird -- `agent.Config.TmpDir` is new (and exported, so people can work with it to set up a new Agent) but `Agent.tmpDir` is not -- I left it that way because I wasn't sure we wanted people messing with an existing agent's temporary directory (they should do that through the `agent.Config` type).

If that's too confusing (or wrong) I'm happy to change it!

Reply:
Ah, I think I looked at the wrong struct when I glanced back up. I think the comment should be `tmpDir` not `tmpdir` (since the "d" is capitalized in the field name and Go is case sensitive).